vslib/src/CorePortIndexMapFileParser.cpp

```diff
@@ -0,0 +1,230 @@
+#include "CorePortIndexMapFileParser.h"
+
+#include "swss/logger.h"
+#include "swss/tokenize.h"
+
+#include <net/if.h>
+
+#include <fstream>
+
+using namespace saivs;
+
+// must be the same or less as SAI_VS_SWITCH_INDEX_MAX
+#define MAX_SWITCH_INDEX (255)
```

Comment:
then use SAI_VS_SWITCH_INDEX_MAX ?

Reply:
SAI_VS_SWITCH_INDEX_MAX is defined in a .cpp file. Not visible here.